src/plugins/my-kiva-homepage-mixin.js

```diff
@@ -0,0 +1,27 @@
+import myKivaForAllUsersQuery from '#src/graphql/query/shared/myKivaForAllUsers.graphql';
+import { getIsMyKivaEnabled, MY_KIVA_FOR_ALL_USERS_KEY } from '#src/util/myKivaUtils';
+import { readBoolSetting } from '#src/util/settingsUtils';
+
+export default {
+	data() {
+		return {
+			redirectToMyKivaHomepage: false,
+		};
+	},
+	async created() {
```

Comment:
Can this be moved to mounted and still work as expected? Otherwise this will run server-side on all page requests in the header.